<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformanceTestWidget</class>
 <widget class="QWidget" name="PerformanceTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Performance Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="mResultLabel">
     <property name="text">
      <string>Result</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mPerformanceTestComboBox">
     <item>
      <property name="text">
       <string>Max Speed @ Alt/Power</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Min Speed @ Alt/Power</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max G-Load @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Sustained G-Load @ Alt/Power</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Level Flight Range @ Alt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Lofted Range @ Alt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Sustained Pitch Angle @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Roll Rate @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time to Damp Roll @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Pitch Rate @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time to Damp Pitch @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Yaw Rate @ Alt/Speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time to Damp Yaw @ Alt/Speed</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="mStartTestButton">
     <property name="text">
      <string>Start Test</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mPerformanceTestLabel">
     <property name="text">
      <string>Performance Test</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mResultLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="mTestProgressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="mVarOneLineEdit">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mVarOneLabel">
       <property name="text">
        <string>Var1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="mVarTwoLineEdit">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="mVarTwoLabel">
       <property name="text">
        <string>Var2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
